kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/stats/PrometheusStatsLogger.java

```diff
@@ -78,6 +77,22 @@ private ScopeContext scopeContext(String name) {
     }
 
     private String completeName(String name) {
-        return Collector.sanitizeMetricName(scope.isEmpty() ? name : Joiner.on('_').join(scope, name));
```

Comment:
Can we upgrade the `prometheus:simpleclient ` version?

Reply:
The dependency is from Pulsar. If we choose to upgrade the `simpleclient`, we have to import the dependency specially. Not sure if there are API compatibility issues.

Regarding the Pulsar side, the dependency cannot be cherry-picked into 2.10.x or earlier, see https://github.com/apache/pulsar/pull/13785#issuecomment-1235189724